extensions/documents/homie_last_updated_extension.md

```diff
@@ -0,0 +1,73 @@
+# Last-updated timestamp
+
+Version: **<!--VERSION-->1.0.0<!--VERSION-->**
+Date: **<!--DATE-->25. Jan 2020<!--DATE-->**
+Authors: **<!--AUTHORS-->Max Berger<!--AUTHORS-->**
+License: **<!--LICENSE-->[CCA 4.0](https://homieiot.github.io/license)<!--LICENSE-->**
+
+## Abstract
+This extension adds best-effort last-updated timestamps to properties.
+
+Homie devices will publish a last-updated timestamp along with any property value changes. The last-update timestamp can be used by devices or gateways that do not constantly pull data from the real back-end. In this case, they can report when they last actually updated the data, so that clients can act accordingly, and maybe ignore old data, or display this information in an UI.
+## Homie Version
+This extension supports Homie `3.0.1` and `4.x`.
+
+## Extension Identifier
```

Comment:
There is something wrong with your extension identifier.  Assuming you intended the ID `name.berger.name.last-updated`, which seems a bit odd to me, the  **$extensions entry** should be `name.berger.name.last-updated:1.0.0:[3.0.1;4.x]`. 

I would suggest using a more convenient ID in domain format, like [your country code].berger.last-updated, but thats up to you :)

Reply:
Apologies, there was indeed a spelling mistake here. It should be name.berger.max in order to follow the convention of matching website-owner to id-owner.  (My personal website is https://max.berger.name/ ).

Of course, we could also use the homie identifier, but then this would seem like an "official" extension. Would that be preferred?

Reply:
Very often the domain registrar is actually used to identify a purpose like "org" or "io" for public use / invented by public organisations or individuals vs "com" for commercial invented things or used purposes. That's how I see it used in the Java world at least. The reverse domain doesn't always actually exist.

Because the "name" TLD is not common I wouldn't use it here, as it is hardly identified as being a TLD. I suggest something like "org.berger.last-updated" or similar.

Reply:
changed id again